http-client/src/main/java/org/triplea/http/client/ErrorReportingHttpClient.java

```diff
@@ -0,0 +1,57 @@
+package org.triplea.http.client;
```

Comment:
:+1: Seems like you've standardized the package names in these two new projects.  Will there be a follow-up PR to update the `http-server` project to use `org.triplea.http.server` as the root package?

Reply:
Good question, I'm not sure. I'm open to suggestions on this one. Here is how it looks for me:

![screenshot from 2018-09-03 19-56-33](https://user-images.githubusercontent.com/12397753/45007633-875fa800-afb3-11e8-947b-4288228a2050.png)

In the above, I like how `http` splits the http server pieces away from the business/domain processing. Anything not in the `http` layer will have no dependency on that, http talks to `reporting.error` by passing it objects from `http-data` (but in this case the API is still a `String`, it's still yet to be updated and is the simplistic initial server API)

Another way to look at it, I don't want `org.triplea.server.reporting.error.upload` to have `http` in it anywhere:
- would couple to one server implementation or another, so replacing spark with dropwizard might not be simple if we did that
- help keep some business layer processing purity; keep the processing domain split from http handling.


Should:  `org.triplea.server.reporting.error.upload`  be updated in terms of package name?
Second question, where would the `http.spark` and `http.dropwizard` branches start?